platform/src/main/java/org/stellar/anchor/platform/utils/RpcUtil.java

```diff
@@ -48,7 +55,7 @@ public static void validateRpcRequest(RpcRequest rpcRequest) throws InvalidReque
     List<String> messages = new ArrayList<>();
     if (!JSON_RPC_VERSION.equals(rpcRequest.getJsonrpc())) {
       messages.add(
-          String.format("Unsupported JSON-RPC protocol version[%s]", rpcRequest.getJsonrpc()));
+          String.format("Unsupported JSON-RPC protocol version [%s]", rpcRequest.getJsonrpc()));
```

Comment:
Do we need this space before square brackets? It is ok for me, but I just don't like, that we have different log formats